.github/instructions/uv-python-usage-guide.instructions.md

```
@@ -0,0 +1,798 @@
+cd my-project
+uv init
+```
+
+This creates:
+- `pyproject.toml` - Project configuration and dependencies
+- `.python-version` - Python version specification
+- `README.md` - Project documentation
+- `main.py` - Initial Python file
+
+### Project Structure
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)